src/debug_backend.md

```diff
@@ -74,32 +83,43 @@ However, for longer errors including `(` or `)` you will need to escape them cor
 If all works out, you will see a lot of iterations, ending with a new `reduced.ll` file. 
 Verify with `opt` that you still get the same error.
 
+### Advanced Debugging: Manual LLVM-IR Investigation
+
+Once you have a minimized reproducer (`mwe.ll` or `reduced.ll`), you can delve deeper:
+
+*   **Manual Editing:** Try manually rewriting the LLVM-IR. For certain issues, like those involving indirect calls, you might investigate Enzyme-specific intrinsics like `__enzyme_virtualreverse`. Understanding how to use these might require consulting Enzyme's documentation or source code.
+*   **Enzyme Test Cases:** Look for relevant test cases within the [Enzyme repository](https://github.com/EnzymeAD/Enzyme/tree/main/enzyme/test) that might demonstrate the correct usage of features or intrinsics related to your problem.
+
 ## 5) Report your bug.
 
-Afterwards, you should be able to copy and paste your `mwe.ll` (and `reduced.ll`) example into our [compiler explorer](https://enzyme.mit.edu/explorer/).  
+Afterwards, you should be able to copy and paste your `mwe.ll` (or `reduced.ll`) example into the [Enzyme compiler explorer](https://enzyme.mit.edu/explorer/).  
 Select `LLVM IR` as language and `opt 20` as compiler. Replace the field to the right of your compiler with `-passes="enzyme"`, if it is not already set. 
 Hopefully, you will see once again your now familiar error. Please use the share button to copy links to them.
 
-Please create an issue on [https://github.com/EnzymeAD/Enzyme/issues](github) and share `mwe.ll` and (if you have it) `reduced.ll`, as well as links to the compiler explorer. Please feel free to also add your Rust code or a link to it. With that, hopefully someone from the Enzyme core repository will be able to fix your bug. Once that happened, I will update the Enzyme submodule inside the rust compiler, which should allow you to now differentiate your Rust code. Thanks for helping us to improve Rust-AD.
+Please create an issue on [https://github.com/EnzymeAD/Enzyme/issues](https://github.com/EnzymeAD/Enzyme/issues) and share `mwe.ll` and (if you have it) `reduced.ll`, as well as links to the compiler explorer. Please feel free to also add your Rust code or a link to it.
+
+**Documenting Findings:** Some Enzyme errors, like `"Attempting to call an indirect active function whose runtime value is inactive"`, have historically caused confusion. If you investigate such an issue, even if you don't find a complete solution, please consider documenting your findings (e.g., in the Enzyme issue or by proposing an update to these docs). This helps prevent others from starting from scratch.
+
+With a clear reproducer and documentation, hopefully someone from the Enzyme core repository will be able to fix your bug. Once that happens, the Enzyme submodule inside the rust compiler will be updated, which should allow you to differentiate your Rust code. Thanks for helping us to improve Rust-AD.
```

Comment:
Nit: EnzymeAD/Enzyme isn’t called Enzyme core, perhaps rephrase “someone from the Enzyme core repository” => “an Enzyme developer”